python/python/glide/async_commands/core.py

```diff
@@ -132,20 +132,124 @@ class InfBound(Enum):
     NEG_INF = "-inf"
 
 
-class ScoreLimit:
+class LexInfBound(Enum):
     """
-    Represents a score limit in a sorted set.
+    Enumeration representing lexicographic positive and negative infinity bounds for sorted set scores.
+    """
+
+    POS_INF = "+"
+    NEG_INF = "-"
+
+
+class ScoreBoundary:
+    """
+    Represents a score boundary in a sorted set.
```

Comment:
Represents a specific score boundary in a sorted set.

Reply:
changing that to numeric score boundry